lib/vis_milk2/CMakeLists.txt

```diff
@@ -56,9 +71,21 @@ set(SOURCES dx11/CommonStates.cpp
             vis_milk2/texmgr.cpp
             vis_milk2/utility.cpp)
 
+if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+  set(SOURCES ${SOURCES} 
+    ns-eel2/asm-nseel-x64.asm
+    ns-eel2/glue_x86_64.h
+  )
+else()
+  set(SOURCES ${SOURCES} 
+    ns-eel2/asm-nseel-x86-msvc.c
```

Comment:
don't add `asm-nseel-x86-msvc.c` to sources. this breaks win32 build. 
`nseel-cfunc.c` is included by `nseel-cfunc.c`

Reply:
ahh, didn't notice that